ui/admin/tests/integration/components/scope-picker/index-test.js

```diff
@@ -0,0 +1,196 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'admin/tests/helpers';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import { click } from '@ember/test-helpers';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+import { setupIntl } from 'ember-intl/test-support';
+
+module('Integration | Component | scope-picker/index', function (hooks) {
+  setupRenderingTest(hooks);
+  setupMirage(hooks);
+  setupIntl(hooks, 'en-us');
+
+  const SCOPE_PICKER_DROPDOWN = '.scope-picker button';
```

Comment:
Nice work!

Wondering, as a general rule, should we move these constants into their own selectors file? I am noticing this trend throughout our test suite.

Reply:
great question! i checked in with @cameronperera about this. currently we only have the separate files for selectors in acceptance tests for admin ui. There needs to be some planning done around how to handle selectors in other test folders. For now we'll leave these as is.

Reply:
Indeed! we will came back to the integration tests once we finish with acceptance 😉 let's keep the current pattern as it is till then.